Update to 0.6.2

ptang-vn opened the pull request:
# Description
Updated the project to support 0.5.0+

There will be one more commit that will add modal use cases in the "More" menu


![device-2019-11-20-220351](https://user-images.githubusercontent.com/55396845/69300608-b9bdeb00-0be1-11ea-96a1-2fa828b7cc43.png)


## Ticket Number
[T1991](https://airtable.com/tbl4lm32cBJ1JL2IG/viwo46AGJOIrceTYh/recsPXZWeLYvI4ti2?blocks=bipoc1l7J0oLHmrgf)

## PR Checklist
- [x] Add at least 2 people as Reviewer
- [x] Add at least 1 person as Assignee
- [ ] Ensure Pull Request passes all checks
- [ ] Add unit tests where possible
- [x] Ensure code confirms to [VenueNext coding standards](https://github.com/venuenext/venuenext-standards/tree/master/android)

ptang-vn committed:
Added maven publish script included by build.gradle

Integrators should populate their JFrog credentials here for SDK access

ptang-vn committed:
Added VN SDK config XML as a central point for configuring the SDK

ptang-vn committed:
Added support for v0.5.0 of the SDK

- Added Wallet
- Added Ticketmaster Presence SDK integration
- Fixed coloring issues with bottom navigation
- Added deep linking support for experience transfers and virtual currency transfers
- Updated color scheme

Dmedina-VN commented on app/src/main/java/com/venuenext/venuenextsdkdemo/MainActivity.kt:
```diff
@@ -1,168 +1,172 @@
+                navController.navigate(R.id.action_to_food_bev_flow)
+                true
+            }
+            // This flow is not currently supported by the SDK
```
why keep this dead code here then?

ptang-vn replied:
I can remove it

ptang-vn committed:
Added support for 0.6.0

- Added modal support of VN flows via settings
- Added Ticketmaster improvements for logging out
- Added network security config for proxy testing
- Showing back button in VN flows disabled by default (since we show non-modal first)

Note: Settings adapters and sections will be flattened later

ptang-vn committed:
Reverted centralized key mappings from XML

ptang-vn committed:
Updated to 0.6.1

ptang-vn committed:
Stability updates for 0.6.1

- Replaced Ticketmaster view with a logout button
- No longer showing Wallet unless the user is logged in

ptang-vn committed:
Updated to 0.6.2

- Added app version (SDK version) to the settings screen
